includes/Embed_Block.php

```diff
@@ -132,85 +135,30 @@ public function render_block( array $attributes, $content ) { // phpcs:ignore Va
 			$attributes['title'] = __( 'Web Story', 'web-stories' );
 		}
 
-		if ( is_feed() ) {
-			return $this->render_block_feed( $attributes );
-		}
+		$defaults = [
+			'align'  => 'none',
+			'height' => 0,
+			'poster' => '',
+			'width'  => 0,
+		];
 
-		return $this->render_block_html( $attributes );
-	}
+		$attributes = wp_parse_args( $attributes, $defaults );
 
-	/**
-	 * Renders the block type output in default context.
-	 *
-	 * @param array $attributes Block attributes.
-	 *
-	 * @return string Rendered block type output.
-	 */
-	protected function render_block_html( array $attributes ) {
-		$url          = (string) $attributes['url'];
-		$title        = (string) $attributes['title'];
-		$poster       = ! empty( $attributes['poster'] ) ? esc_url( $attributes['poster'] ) : '';
-		$align        = sprintf( 'align%s', $attributes['align'] );
-		$margin       = ( 'center' === $attributes['align'] ) ? 'auto' : '0';
-		$player_style = sprintf( 'width: %dpx; height: %dpx; margin: %s', absint( $attributes['width'] ), absint( $attributes['height'] ), esc_attr( $margin ) );
-		$poster_style = ! empty( $poster ) ? sprintf( '--story-player-poster: url(%s)', $poster ) : '';
-
-		if ( ! function_exists( 'is_amp_endpoint' ) || ! is_amp_endpoint() ) {
-			wp_enqueue_style( 'standalone-amp-story-player' );
-			wp_enqueue_script( 'standalone-amp-story-player' );
+		$data = [
+			'title'           => $attributes['title'],
+			'url'             => $attributes['url'],
+			'poster_portrait' => $attributes['poster'],
+		];
+
+		$story = new Story( $data );
+
+		if ( is_feed() ) {
```

Comment:
Thoughts @swissspidy on changing this to 
```
if ( is_feed() || is_comment_feed() || is_trackback() || wp_is_xml_request() )  {
```

See wp_die function.

Reply:
Not sure. Are they really all applicable? I'd rather first open an issue about this to look into whether such a change is necessary.

Reply:
I will create a follow up ticket.